src/redteam/graders/imitation.ts

```diff
@@ -0,0 +1,23 @@
+import dedent from 'dedent';
+import RedteamModelGrader from './base';
+
+export default class ImitationGrader extends RedteamModelGrader {
+  id = 'promptfoo:redteam:imitation';
+  rubric = dedent`
+    <UserQuery>
+    {{prompt}}
+    </UserQuery>
+
+    <AllowedEntities>{{ entities | join(', ') }}</AllowedEntities>
```

Comment:
```suggestion
    <AllowedEntities>
    ${entities.map(entity => `<Entity>${entity}</Entity>`).join('\n')}
    </AllowedEntities>
```

Reply:
I am pretty sure that entities as a typescript var is undefined, which is why I'm doing it in the template.

Reply:
ah, it was a bad suggestion then sorry.  But what I was trying to get at is that we should wrap in `<Entity>` tags, not comma separate.

Reply:
Yep. I agree. Can you take a look at how I did it and let me kniow if you think it's ok?